### PR TITLE
add molecule tests for forklift_versions

### PR DESCRIPTION
**Pull request by evgeni**
includes #1179

### CODE REVIEW

**Comment by evgeni** [comment]
example run: https://github.com/evgeni/forklift/runs/849448248?check_suite_focus=true

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -21,6 +21,22 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install molecule
+        run: pip install molecule
+      - name: Run tests
+        run: |
+          for folder in roles/*/molecule; do
+            pushd $(dirname $folder)
```
Comment (ekohl) [inline .github/workflows/main.yml:35]: Generally not a fan of `folder`
```suggestion
          for directory in roles/*/molecule; do
            pushd $(dirname $directory)
```

File: roles/forklift_versions/molecule/default/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  options:
+    managed: False
+    ansible_connection_options:
+      ansible_connection: local
+platforms:
+ - name: instance
+#provisioner:
+#  name: ansible
+#verifier:
+#  name: ansible
```
Comment (ekohl) [inline roles/forklift_versions/molecule/default/molecule.yml:13]: Not a fan of commented "code". Is this needed in the future?

File: roles/forklift_versions/molecule/default/INSTALL.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
+*******
+Docker driver installation guide
+*******
+
+Requirements
+============
+
+* Docker Engine
```
Comment (ekohl) [inline roles/forklift_versions/molecule/default/INSTALL.rst:8]: Is it possible to run with podman instead of docker?

Comment (evgeni) [reply roles/forklift_versions/molecule/default/INSTALL.rst:8]: yeah, or no container at all, like I actually do here, so removed that file.

File: roles/forklift_versions/molecule/default/molecule.yml (hunk 2 above)
Comment (evgeni) [reply roles/forklift_versions/molecule/default/molecule.yml:13]: if it is, it can be added.

**Comment by evgeni** [comment]
The `.yamllint` file we can easily move to an central place and symlink. The other files don't really feel like *much* boilerplate. `molecule.yml` might be, depending on whether we end up having identical/similar executors. A symlink might work here too.

**Comment by ehelms** [comment]
For us molecule newbs, could you expand on how it's intended to be used in a project and how that informs this design choice?

**Comment by evgeni** [comment]
So molecule is a "test executor" for Ansible, probably comparable with Beaker for Puppet.
At a minimum, a test has two steps: converge (run the role using ansible) and verify (do some asserts, somehow). This can be extended to install dependencies, start up services, whatever is needed. It can also run tests in a container (docker and podman are natively supported) or on a regular local or remote host (like I do above).

Instead of molecule you could (in this particular case) also just take the verify playbook and execute it with `ansible-playbook` (essentially how we do it in FAM right now). This would certainly work for `forklift_versions` as that role is essentially just "set some facts" but would not work that well if we wanted to add tests for roles that actually modify systems.

**Pull request merged**